tox.ini

```diff
@@ -174,4 +174,11 @@ norecursedirs = system_tests examples .* build dist CVS _darcs {arch} *.egg venv
 junit_suite_name = nimi-python
 python_files = *.py
 junit_family = xunit1
-
+markers = # Defines custom markers used by nidcpower system tests. Prevents PytestUnknownMarkWarning.
```

Comment:
This tox.ini is not used for running system tests, does this actually get rid of the warning?!

This code-generated tox.ini is what is used for running system tests on nimi-bot agents: https://github.com/ni/nimi-python/blob/master/generated/nidcpower/tox-system_tests.ini

Reply:
I've moved the custom markers to the mako template in https://github.com/ni/nimi-python/pull/1624/commits/7b97e591e765b91e2a0e785448994a75aaf30398